docs/src/connection_pooling.md

```diff
@@ -256,6 +256,36 @@ spec:
               memory: 500Mi
 ```
 
+## Service Template
+
+Sometimes, your pooler will require some different labels, annotations, or even change
+the type of the service, you can achive that by using the `serviceTemplate` field:
+
+```yaml
+apiVersion: postgresql.cnpg.io/v1
+kind: Pooler
+metadata:
+  name: pooler-example-rw
+spec:
+  cluster:
+    name: cluster-example
+  instances: 3
+  type: rw
+  serviceTemplate:
+    metadata:
+      labels:
+        app: pooler
+    spec:
+      type: LoadBalancer
+  pgbouncer:
+    poolMode: session
+    parameters:
+      max_client_conn: "1000"
+      default_pool_size: "10"
+```
+
+An example of this is in [`pooler-external.yaml`](samples/pooler-external.yaml)
```

Comment:
there's a full section of samples in samples.md where this should go

Reply:
Made that change, thanks!